benchopt/plotting/plot_objective_curve.py

```diff
@@ -195,14 +167,17 @@ def get_solver_color(solver):
     return color
 
 
-def get_solver_marker(solver):
+def get_solver_marker(solver, plotly=True):
     if solver in html_solver_styles and 'marker' in html_solver_styles[solver]:
         return html_solver_styles[solver]['marker']
```

Comment:
There is something wrong with the logic... here, the marker `0` will never be used, and the color `1` neither ^^.
This is due to the original design of this function but could we merge the two into a `get_solver_style` that generate both color and marker at the same time? and then select wathever we need.